### PR TITLE
Add api types to toc

### LINKED ISSUE #1450: API documentation is not accessible from the documentation site's sidebar
There does not appear to be a link to the API documentation from the documentation site's sidebar.

### PR DESCRIPTION
**Pull request by carlisia**
Fixes #1450. 

I'll add a PR for the `docs` folder too.

image

Signed-off-by: Carlisia <carlisiac@vmware.com>

### CODE REVIEW

File: master/api-types/README.md [hunk 1]
```diff
@@ -6,5 +6,9 @@ Here we list the API types that have some functionality that you can only config
 (hooks)
 
 * [Backup][1]
+* [BackupStorageLocation][2]
+* [VolumeSnapshotLocation][3]
 
 [1]: backup.md
+[2]: backupstoragelocation.md
+[3]: volumesnapshotlocation.m
```
Comment (skriss) [inline master/api-types/README.md]: `.m` -> `.md`

File: v0.11.0/api-types/README.md [hunk 2]
```diff
@@ -6,5 +6,9 @@ Here we list the API types that have some functionality that you can only config
 (hooks)
 
 * [Backup][1]
+* [BackupStorageLocation][2]
+* [VolumeSnapshotLocation][3]
 
 [1]: backup.md
+[2]: backupstoragelocation.md
+[3]: volumesnapshotlocation.m
```
Comment (skriss) [inline v0.11.0/api-types/README.md]: `.m` -> `.md`

File: v1.0.0-rc.1/api-types/README.md [hunk 3]
```diff
@@ -6,5 +6,9 @@ Here we list the API types that have some functionality that you can only config
 (hooks)
 
 * [Backup][1]
+* [BackupStorageLocation][2]
+* [VolumeSnapshotLocation][3]
 
 [1]: backup.md
+[2]: backupstoragelocation.md
+[3]: volumesnapshotlocation.m
```
Comment (skriss) [inline v1.0.0-rc.1/api-types/README.md]: `.m` -> `.md`

**Comment by skriss** [comment]
We're probably going to end up having to redo this once the new site PR goes in, so we may just want to hold off until then.

**Comment by carlisia** [comment]
Maybe they'll end up rebasing? Either way is fine.

**Comment by skriss** [comment]
so...unfortunately you'll have to redirect this to master, to the `site/docs` dir

**Comment by carlisia** [comment]
Np!

**Comment by carlisia** [comment]
Closing in lieu of https://github.com/heptio/velero/pull/1496.

**Pull request closed without merging**